pages/resources/user.mdx

```diff
@@ -450,6 +450,92 @@ How a user's name gets displayed, such as font, colors, gradient, glow.
 | 5     | POP      | Colored dropshadow                  |
 | 6     | GLOW     | Alternate gradient style            |
 
+## Game widgets
+
+#### Game Widgets Terminology Specification
+This section may contain terminology which can easily get mixed up. This list specifies their definition.
+
+  - Game Widget: A game widget is a single widget list (e.g. favourite games, games in rotation) containing Game objects.
+  - Game/Game Object: A "Game" is in this case referring to a game within a game widget list, containing an optional comment and tags.
+  - Detailed Game Widget: A detailed game widget is a game widget where the games in the list can display tags and a comment.
+  - Application Game Widget: An application game widget is a special kind of game widget which displays special data/statistics specific to a supported game.
+
+### Game Widget object
+
+Contains a data field to provide the data of the widget.
+
+| Field       | Type                                            | Description                                                                                                                          |
+|-------------|-------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------|
+| data        | [game widget data](#game-widget-data-structure) | Data field containing the data of the game widget.                                                                                   |
```

Comment:
```suggestion
| data        | [game widget data](#game-widget-data-structure) object | The data of the game widget                                                                                   |
```
- object data types should be followed by "object"
- no periods in field/enum descriptions (also keep em concise xd)